src/components/DiffView/index.spec.tsx

```diff
@@ -656,6 +675,100 @@ describe(__filename, () => {
     expect(root.find(`.${styles.highlightingDisabled}`)).toHaveLength(0);
   });
 
+  it('trims diffs that are too large to display', () => {
```

Comment:
Nit: I think it's fine, but this really tests multiple things (i.e., it checks that the diff was trimmed, it checks that the content is faded, and it checks that the messages about trimming appear). It's probably okay, but I've been told in the past not to test multiple things in a single test as it could make it a bit more difficult to debug when one of those things breaks but not another.

Reply:
Well, the pieces are all connected here. It needs to get the `fadadable` component before it can check that the trimmed content was placed inside. I will update the spec.